extensions/amp-date-picker/1.0/amp-date-picker.css

```diff
@@ -0,0 +1,31 @@
+@import '../../../third_party/react-day-picker/style.css';
+
+.amp-date-picker-calendar-container * {
```

Comment:
Even with modified CSS, this component doesn't look and behave exactly like the original. It's probably worth discussing how much we care about maintaining exact visual parity with different underlying components.

Reply:
I don't feel like having some styling changes is a big deal. As long as most of the primary functionality is there.